diff --git a/node_modules/node-edge-tts/dist/drm.js b/node_modules/node-edge-tts/dist/drm.js
index a45d247..82d0f11 100644
--- a/node_modules/node-edge-tts/dist/drm.js
+++ b/node_modules/node-edge-tts/dist/drm.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.generateSecMsGecToken = exports.TRUSTED_CLIENT_TOKEN = exports.CHROMIUM_FULL_VERSION = void 0;
 const node_crypto_1 = require("node:crypto");
-exports.CHROMIUM_FULL_VERSION = '130.0.2849.68';
+exports.CHROMIUM_FULL_VERSION = '143.0.3650.75';
 exports.TRUSTED_CLIENT_TOKEN = '6A5AA1D4EAFF4E9FB37E23D68491D6F4';
 const WINDOWS_FILE_TIME_EPOCH = 11644473600n;
 function generateSecMsGecToken() {
diff --git a/node_modules/node-edge-tts/dist/edge-tts.js b/node_modules/node-edge-tts/dist/edge-tts.js
index 88af232..e7d11b2 100644
--- a/node_modules/node-edge-tts/dist/edge-tts.js
+++ b/node_modules/node-edge-tts/dist/edge-tts.js
@@ -23,7 +23,7 @@ class EdgeTTS {
             host: 'speech.platform.bing.com',
             origin: 'chrome-extension://jdiccldimpdaibmpdkjnbmckianbfold',
             headers: {
-                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36 Edg/130.0.0.0',
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/143.0.0.0 Safari/537.36 Edg/143.0.0.0',
             },
             agent: this.proxy ? new https_proxy_agent_1.HttpsProxyAgent(this.proxy) : undefined
         });
